### DISCUSSION TITLE
Engined 20% at Idle

### DISCUSSION THREAD

**Discussion by avrunner01** [flybywiresim/aircraft#2835] [Q&A] [answered]
Is 20% at idle correct? I see the baseline A321 is also set at 20%. 
Note: When taxing the aircraft over-speeds and goes above 20 knots very easily.

**Comment by avrunner01** [comment]
I understand the 321NX aircraft engines produce more thrust.

**Reply by hotshotp** [reply]
yes this is correct for both things that you said in your post - a321 also has the same idle at around 18-20%

**Comment by ghost** [comment, marked as answer]
Yes, most jet engines idle around this value, it is impossible for it to idle at 0 because the engine would just be off. Its a bit like a car engine, it does not idle at 0 RPM. Also like a car engine idling, an idling jet will still produce thrust. In the a320, this is enough to move the plane, and if you are light enough, it will move pretty rapidly.
